tools/text_processing/split_file_to_collection/split_file_to_collection.xml

```diff
@@ -396,7 +418,7 @@
                 <element name="test_000001" file="test_1.fastq" ftype="fastq"/>
             </output_collection>
         </test>
-        <!-- splitting of fasta w random assignment and specific filename prefix -->
+        <!-- 12 splitting of fasta w random assignment and specific filename prefix -->
         <test>
```

Comment:
you could enhance all test by using num_expect_outputs https://docs.galaxyproject.org/en/latest/dev/schema.html#id78

Reply:
Cool, thanks! I will update the test to use this!